Resources/Prototypes/Entities/Mobs/NPCs/hivebot.yml

```diff
@@ -0,0 +1,117 @@
+# Base hivebot
+- type: entity
+  parent: [ BaseSimpleMob, MobCombat, MobRobotic ]
+  id: MobHivebot
+  name: hivebot
+  description: Agitating, mechanized pests.
+  components:
+  - type: HTN
+    rootTask:
+      task: SimpleHostileCompound
+    blackboard:
+      NavSmash: !type:Bool
+        true
+  - type: NpcFactionMember
+    factions:
+    - SimpleHostile
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Aliens/hivebot.rsi
+    layers:
+    - map: [ "enum.DamageStateVisualLayers.Base" ]
+      state: basic_attack
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.50
+        density: 80
+        mask:
+        - MobMask
+        layer:
+        - MobLayer
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      20: Dead
+  - type: SlowOnDamage
+    speedModifierThresholds:
+      10: 0.7
+  - type: MeleeWeapon
+    altDisarm: false
+    angle: 0
+    soundHit:
+      path: /Audio/Weapons/bladeslice.ogg
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Slash: 6 # Pretty whatever on their own, but their strength is in numbers
+  - type: Tag
+    tags:
+      - DoorBumpOpener
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 1.5
+    baseSprintSpeed: 2.5
+  - type: PointLight
+    radius: 1.0
```

Comment:
you should set `castShadows` to false here just because these are meant to be swarms and can potentially spawn a fair number of them